stacks/spica/src/bucket/pages/add/add.component.ts

```diff
@@ -77,7 +89,20 @@ export class AddComponent implements OnInit {
     this.data = await this.bhs.revertTo(this.bucketId, this.data._id, historyId).toPromise();
   }
 
+  scheduleOnTrue() {
+    this.scheduleOn = true;
```

Comment:
rename to `scheduledTo `.
Fill that property like this `this.scheduleOn = new Date()`